src/test/java/io/cdap/plugin/gcp/bigquery/sqlengine/builder/BigQueryWindowsAggregationSQLBuilderTest.java

```diff
@@ -44,16 +48,17 @@ public void setUp() {
 
     windowFrameType = WindowAggregationDefinition.WindowFrameType.NONE;
     fullDefinition = WindowAggregationDefinition.builder().select(selectFields).partition(partitionFields)
-      .orderBy(orderFields).windowFrameType(windowFrameType).unboundedFollowing(true).unboundedPreceding(true)
-        .build();
+      .aggregate(aggregationFields).orderBy(orderFields).windowFrameType(windowFrameType).unboundedFollowing(true)
+      .unboundedPreceding(true).build();
```

Comment:
can you also add a test for the ase where preceeding/following are not unbounded?

Reply:
Done